ql-qlf-plugin/synth_quicklogic.cc

```diff
@@ -225,6 +225,7 @@ struct SynthQuickLogicPass : public ScriptPass {
                 run("shregmap -minlen 8 -maxlen 8");
             }
             if (family == "qlf_k6n10") {
+                run("async2sync");
```

Comment:
I guess we shouldn't use the `async2sync` pass as it caused us some trouble in the past. This should be done as it is for `qlf_k4n8` with `dfflegalize` (a few lines below)